doc/source/development/contributing.rst

```diff
@@ -649,9 +649,13 @@ Thus, good style is a requirement for submitting code to pandas.
 There is a tool in pandas to help contributors verify their changes before
 contributing them to the project::
 
-   ./ci/code_checks.sh
+   pre-commit run --all-files
 
-The script verifies the linting of code files, it looks for common mistake patterns
+or, to only check files which were modified::
+
+   pre-commit run --from-ref=upstream/master --to-ref=HEAD --all-files
+
+The command verifies the linting of code files, it looks for common mistake patterns
```

Comment:
sorry, my bad, this next paragraph refers specifically to `./ci/code_checks.sh`, so that still needs updating...do you want to open a new issue to remind ourselves to update `doc/source/development/contributing.rst`, revert all changes to this file in this PR so that this PR only modifies .github/PULL_REQUEST_TEMPLATE.md, and then we can get it in?

Reply:
Yes I would like to do that.